src/cmd/cmdpkg_create.rb.in

```diff
@@ -0,0 +1,196 @@
+      end
+      opts.on('-d','--ign_dependecies', String, 'Package depends on common ign packages') do
+        options['ign_dependecies'] = 1
+      end
+      opts.on('-g [arg]', '--gazebo_version [arg]',String, 'Gazebo version name for dependencies') do |g|
+        options['gazebo_version'] = g
+      end
+      
+    end # opt_parser do
+    begin
+      opt_parser.parse!(args)
+    rescue
+      puts usage
+      exit(-1)
+    end
+
+    # Check if all options are correct
+
+    if !(options.key?('name') && options['name'].class()==String)
+     puts "ERROR: Please provide proper package name.\n"
+     puts usage
+     exit(-1)
+    end
+    if (options.key?('built_type') && !(['ament_cmake','cmake'].include? options["built_type"]))
+      puts "WARNING: Invalid Built-type,defaulting to cmake\n"
+      options['built_type'] = "cmake"
+    end
+    if (options.key?('export_type') && !(['plain_cmake','colcon'].include? options["export_type"]))
+      puts "WARNING: Invalid Export-type,defaulting to colcon\n"
+      options['export_type'] = "colcon"
+    end
+    if (options.key?('gazebo_version') && !(['acropolis','blueprint','citadel','dome','edifice','fortress','garden'].include? options["gazebo_version"]))
+      puts "WARNING: Invalid Gazebo Version,defaulting to fortress\n"
+      options['gazebo_version'] = "fortress"
+    end
+    #Giving default values
+
+    if !options.key?('init-with-example-system')
+      options['init-with-example-system'] = 0
+    end 
+    if !options.key?('built_type')
+      options['built_type'] = "cmake"
+    end 
+    if !options.key?('export_type')
+      options['export_type'] = "colcon"
+    end 
+    if !options.key?('ign_dependecies')
+      options['ign_dependecies'] = 0
+    end
+    if !options.key?('gazebo_version')
+      options['gazebo_version'] = "fortress"
+    end
+
+    options['command'] = args[0]
+
+    options
+  end # parse()
+
+  def execute(args)
+    options = parse(args)
+
+    # Read the plugin that handles the command.
+    if LIBRARY_NAME[0] == '/'
+      # If the first character is a slash, we'll assume that we've been given an
+      # absolute path to the library. This is only used during test mode.
+      plugin = LIBRARY_NAME
+    else
+      # We're assuming that the library path is relative to the current
+      # location of this script.
+      plugin = File.expand_path(File.join(File.dirname(__FILE__), LIBRARY_NAME))
+    end
+    conf_version = LIBRARY_VERSION
+
+
+
+    ##Code for package generation
+    if Dir.exists?(options['name'])
+      puts "ERROR:Folder with this name already exists."
+      exit(-1)
+    end
+
+    #Creates Package folder
+    Dir.mkdir(options['name'])
+
+    #Fetching dependencies versions for common dependencies
+    if options['ign_dependecies']==1
+      dependencies = URI.open('https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-%s.yaml'% options['gazebo_version']){|f| f.read}
```

Comment:
This is now at gazebo-tooling if Im not wrong. https://github.com/gazebo-tooling/gazebodistro